client/src/Components/RatingsAndReviews/ReviewList/ReviewTiles.jsx

```diff
@@ -22,27 +26,41 @@ var ReviewTiles = (props) => {
     }
   }
 
+  var seeMoreClick = (event) => {
+    setReviewBodyRender(props.reviews.body)
+    setSeeMore(false)
+  }
 
 
 
   return (
     <ReviewTile>
    <div>
-    <div>⭐️⭐️⭐️⭐️⭐️ Star Rating</div>
-    <div>Date Of Review: {formatDate(props.reviews.date)}</div>
-    <div>Summary: {props.reviews.summary.substr(0, 60)}</div>
-    <div><b>Review Body: {props.reviews.body.substr(0, 250)}</b>
+     <TopContainer>
+        <div>⭐️⭐️⭐️⭐️⭐️</div>
+        <UserandDate>
+          <span>{props.reviews.reviewer_name}   </span>
+          <span> {`,  ${formatDate(props.reviews.date)}`}</span>
+        </UserandDate>
+      </TopContainer>
+    <div><b>{props.reviews.summary.substr(0, 60)}</b></div>
+    <div>{reviewBodyRender}
+    <div>{props.reviews.body.length > 250 && seeMore && <a onClick={seeMoreClick} style={{cursor: 'pointer'}}>See More</a>}</div>
```

Comment:
I'm assuming that you aim to do conditional rendering here and if I understand correctly, inside this `<div>`, if `reviews.body.length > 250` and `seeMore` is true, the `See More` link is rendered. This should work. Consider putting the surrounding `<div>` wrapper outside of this conditional render inside the conditional render, because even if we don't render the `See More` link, I think the `<div>` will still render and be empty if that makes sense.